crates/rover-client/src/operations/persisted_queries/publish/types.rs

```diff
@@ -107,6 +110,138 @@ impl From<PersistedQueriesPublishInput> for QueryVariables {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct RelayPersistedQueryManifest {
+    #[serde(flatten)]
+    operations: HashMap<String, String>,
+}
+
+impl TryFrom<RelayPersistedQueryManifest> for ApolloPersistedQueryManifest {
+    type Error = RoverClientError;
+
+    fn try_from(relay_manifest: RelayPersistedQueryManifest) -> Result<Self, Self::Error> {
+        if relay_manifest.operations.is_empty() {
```

Comment:
I guess I'm not clear on why we would check this here if we don't for Apollo-format manifests.